tests/posit/connect/oauth/test_oauth.py

```diff
@@ -23,5 +23,6 @@ def test_get_credentials(self):
                 "token_type": "Bearer",
             },
         )
-        con = Client(api_key="12345", url="https://connect.example/")
-        assert con.oauth.get_credentials("cit")["access_token"] == "viewer-token"
+        c = Client(api_key="12345", url="https://connect.example/")
+        c.ctx.version = None
```

Comment:
Should we test here that the oauth client resource respects the server's version constraint?